scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/hvson8.yaml

```diff
@@ -0,0 +1,44 @@
+HVSON-8-1EP_3.25x2.35mm_P0.8mm_E1.1x1.75mm:
```

Comment:
Dimensions are 4x4mm with an EP of 2.2x3.1mm.

Reply:
Yes sorry